### PR TITLE
Log branch

### PR DESCRIPTION
**Pull request by lingwei-gu**
When installing faiss-cpu (in pip installation guide), there were errors when running the unittest.
Fixed the issue by specifying the faiss-cpu version according to this link https://github.com/facebookresearch/faiss/issues/708

Here's the environment
MacOS BigSur version 11.6 (20G165)
MacBook Pro (16-inch, 2019)
Processor: 2.3 GHz 8-Core Intel Core i9
Memory: 16 GB 2667 MHz DDR4

### CODE REVIEW

**Comment by MXueguang** [comment]
Hi @lingwei-gu,
Thanks for the reproduction.

>When installing faiss-cpu (in pip installation guide).

Have you tried install faiss-cpu via conda? i.e. `conda install -c pytorch faiss-cpu`
we usually use faiss-cpu 1.7.0+

For the reproduction, what results did you get? exactly same as the number in documents?

**Comment by lingwei-gu** [comment]
> 



> Hi @lingwei-gu, Thanks for the reproduction.
> 
> > When installing faiss-cpu (in pip installation guide).
> 
> Have you tried install faiss-cpu via conda? i.e. `conda install -c pytorch faiss-cpu` we usually use faiss-cpu 1.7.0+
> 
> For the reproduction, what results did you get? exactly same as the number in documents?

**Comment by lingwei-gu** [comment]
Hi @MXueguang ,
I used conda to activate the environment as stated in the guide, and used `conda install faiss-cpu -c pytorch`

For the reproduction, I got exactly the same results as the documents.

**Comment by MXueguang** [comment]
which python version are you using?

**Comment by lingwei-gu** [comment]
Following the guide, the guide created a environment using python 3.6.

**Comment by lingwei-gu** [comment]
@MXueguang In theory, there shouldn't be problems because all steps are done in the conda environment. Do you want me to revoke the change to installation.md?

**Comment by MXueguang** [comment]
seems to be a similar issue here https://github.com/castorini/pyserini/pull/911.
Could you help with confirming if the issue you got still exist for Python 3.8 (in the conda environment).

If the issue happens in python3.6 only, could you add a note in installation.md mentioning the fix is for python3.6 only?

**Comment by tyao-t** [comment]
> @MXueguang In theory, there shouldn't be problems because all steps are done in the conda environment. Do you want me to revoke the change to installation.md?

@lingwei-gu I think you can leave `conda install -c pytorch faiss-cpu` as is, and destroy your current environment first `conda deactivate` and `conda remove -n pyserini --all`. Finally, run the installation steps all over again but use python 3.8 instead `conda create -n pyserini python=3.8`.

**Comment by lingwei-gu** [comment]
@MXueguang @tyao-t 
Updated the PR mentioning that the fix is for python3.6 only. 
I ran the test again with python3.8, and it passed all the unit tests.
Just let me know if we want to change the overall environment to python=3.8 instead

**Comment by lintool** [comment]
We'll circle back to address #916, but this can be merged.

**Review by lintool — APPROVED** [review]

File: docs/installation.md [hunk 1]
```diff
@@ -53,6 +53,7 @@ Install Faiss based on environment
 ```bash
 $ conda install faiss-cpu -c pytorch
 ```
+Note: Use ```conda install faiss-cpu=1.4.0 -c pytorch``` instead for Python3.6 if encountering bugs
```
Comment (lintool) [inline docs/installation.md:56]:
```suggestion
```

**Review by lintool — APPROVED** [review]

**Pull request merged**